frontend/src/component/Dashboard/MemberList.styl

```diff
@@ -133,7 +137,10 @@
       justify-content flex-end
       & > button
         padding 8px 30px
+        background-color transparent
         cursor pointer
+        &:hover
```

Comment:
it would better to have some (2 or 3) buttons classes that defines all styles for buttons. Apart from positioning.
So that all Tracim buttons only uses theses classes